emcc.py

```diff
@@ -2576,7 +2576,7 @@ def get_full_import_name(name):
       settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE.append('__c_longjmp')
 
   if settings.EXCEPTION_HANDLING:
-    settings.REQUIRED_EXPORTS += ['__trap']
+    settings.REQUIRED_EXPORTS += ['__trap', '__cpp_exception']
```

Comment:
Do all users want this to be exported thought?

Reply:
Maybe not. If we create an option that automatically exports all necessary methods and stuff, this better be there I guess. For the moment if I delete this I need to add this in `test_exception_message`, which can be OK given that it already has tons of imports...

Reply:
Anyway I'm OK either way

Reply:
In general we try not to export what we don't use.. do doing in the test for now makes more sense.

Reply:
No, I don't think that has ever come up before. 

`REQUIRED_EXPORTS` is not something you can control from outside.. its an internal settings.

Reply:
Then how can I do it from the test?

Reply:
You would want to add it to `EXPORTED_FUNCTIONS` (despite its name its used to export all symbols).

Reply:
Done